testingfacade/Docs/QuestionSchema.json

```diff
@@ -0,0 +1,177 @@
+                    },
+                    "multi_choice": {
+                        "description": "Multiple answer_id's are expected and should be used to return a list of strings"
+                    },
+                    "action": {
+                        "description": "Instructions to be carried out by the user, no answer to return"
+                    }
+                }
+
+            ]
+        },
+        "question_id": {
+            "description": "A unique string for each individual question"
+        },
+        "name": {
+            "description": "The test name, identical to the question_id unless a test contains multiple questions"
+        },
+        "description": {
+            "description": "String, a brief description of the test"
+        },
+        "question": {
+            "description": "String, human readable question text containing instructions to carry out the test"
+        },
+        "answers": {
+            "type": "array",
+            "description": "Array containing answer objects to be used to select the correct answer_id to be returned.",
+            "examples": [
+                [
+                    {
+                        "answer_id": "answer_2",
+                        "label": "s3/wright",
+                        "description": "Mock sender 3",
+                        "id": "71ce775c-cdc5-479a-bd29-87c081905ecf",
+                        "answer_str": "s3/wright (Mock sender 3, 71ce775c-cdc5-479a-bd29-87c081905ecf)"
+                    }
+                ]
+            ],
+            "additionalItems": true,
+            "items": {
+                "anyOf": [
+                    {
+                        "description": "Individual answer details, the label, description and id of the relevant resource and an associated answer_id",
+                        "required": [
+                            "answer_id",
+                            "label",
+                            "description",
+                            "id",
+                            "answer_str"
+                        ],
+                        "properties": {
+                            "answer_id": {
+                                "description": "String, unique to each answer within a question. This is the value to be returned for validation."
+                            },
+                            "label": {
+                                "description": "Resource label as might be displayed on the NCuT"
+                            },
+                            "description": {
+                                "description": "Resource description as might be displayed on the NCuT"
+                            },
+                            "id": {
+                                "description": "Resource ID as might be displayed on the NCuT"
+                            },
+                            "answer_str": {
```

Comment:
Rather than `_str`, indicating type, can the property name indicate semantics, that it's for human consumption, in contrast to the `label`, `description` and `id` properties? Perhaps `answer_description`?

Those three program-readable values could be separated into a sub-object, e.g. `resource` (with a common schema via `$ref` as mentioned previously).

Then you could do:

```json
{
    "id": "answer_2",
    "resource": {
        "label": "s3/wright",
        "description": "Mock sender 3",
        "id": "71ce775c-cdc5-479a-bd29-87c081905ecf"
    },
    "description": "s3/wright (Mock sender 3, 71ce775c-cdc5-479a-bd29-87c081905ecf)"
}
```

And all of these properties need schema `type` as well as schema `description`!